sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
This line says that the number in question is ``1*sqrt(2) + 0``. That is the intended meaning. ``Tuple()`` denotes the zero element. (Apparently the obvious contradiction has not come to test so far.)